Sources/glTFSceneKit/GLTFConverter+Geometry.swift

```diff
@@ -105,8 +105,15 @@ extension GLTFAccessor {
     }
 }
 
+fileprivate extension GLTF {
+    //let buffer = glTF.buffers![glTF.bufferViews![bufferView].buffer]
+    func buffer(for bufferView: Int) -> GLTFBuffer {
+        return self.buffers![self.bufferViews![bufferView].buffer]
```

Comment:
No guarantees that buffers or bufferViews are exists. Don't force them. Better to return optional GLTFBuffer and check on availability with `if let` or `guard`